backend/ontario/ontario/__init__.py

```diff
@@ -99,29 +122,21 @@ def composite(self, other: "ImageBuilder") -> "ImageBuilder":
         Composites two images.
         """
 
-        node = Gegl.Node()
-        node.set_property("operation", "gegl:over")
-        self.__parent.add_child(node)
-
-        # Connect the last node to the save node.
-        self.__nodes[-1].connect_to("output", node, "aux")
-        other.__nodes[-1].connect_to("output", node, "input")
-
-        self.__nodes.append(node)
+        # TODO: implement gegl:layer operation
```

Comment:
Could this be an exception? Composite is currently used in the Ontario-Web tests and I'd like this tests to error is this is unimplemented.

Reply:
Composite does not currently exist as an operation, the operation gegl:layer does a similar job, but takes arguments described here: https://gegl.org/operations/gegl-layer.html. I will begin implementing this ASAP